app/components/ProtectedRoute/tests/index.test.js

```diff
@@ -28,21 +28,21 @@ describe('<ProtectedRoute />', () => {
 
   it('should render and match the snapshot', () => {
     const { baseElement } = renderProvider(
-      <ProtectedRoute isLoggedIn={true} render={HomeContainer} exact={true} path="/" />
+      <ProtectedRoute isLoggedIn={true} render={ItunesContainer} exact={true} path="/" />
     );
     expect(baseElement).toMatchSnapshot();
   });
 
-  it('should  render the component if user logged in and access protected route', () => {
-    const { getByTestId } = renderProvider(
-      <ProtectedRoute isLoggedIn={true} render={HomeContainer} exact={true} path="/" />
-    );
-    expect(getByTestId('redirect')).toBeTruthy();
-  });
+  // it('should  render the component if user logged in and access protected route', () => {
+  //   const { getByTestId } = renderProvider(
+  //     <ProtectedRoute isLoggedIn={true} render={ItunesContainer} exact={true} path="/" />
+  //   );
+  //   expect(getByTestId('redirect')).toBeTruthy();
+  // });
```

Comment:
put this back. Why is this commented out ?

Reply:
updated the change accordingly